### PR TITLE
[Rebranding] Update Explorer FE and BE api URLs

### LINKED ISSUE #2011: Rebranding: Update Explorer URLs in CowSwap
**Description**

Update Explorer URLs in CowSwap:
https://github.com/gnosis/cowswap/blob/d8687e5efa6fbaeb30a63d79eebc88286ff43648/.env.production#L67-L70

**Proposal**

Apply the same changes as https://github.com/gnosis/gp-ui/issues/914
But on CowSwap

### LINKED ISSUE #2012: Rebranding: Update backend URLs in CowSwap
**Description**

Update API URLs in CowSwap:
https://github.com/gnosis/cowswap/blob/d8687e5efa6fbaeb30a63d79eebc88286ff43648/.env.production#L59-L64

**Proposal**

Apply the same changes as https://github.com/gnosis/gp-ui/issues/915
But on CowSwap

### PR DESCRIPTION
**Pull request by matextrem**
# Summary

closes #2011
closes #2012  

- Updated Explorer URLs (only `prod` and `barn` environments).
- Updated Explorer api URLs.

### CODE REVIEW

**Review by alongoni — APPROVED** [review]

**Review by henrypalacios — APPROVED** [review]

**Comment by alfetopito** [comment]
Looks good, I just won't merge yet as I'm pending on confirmation from DevOps regarding the api.cow.fi domains. They don't seem to be active yet.
Will let you know as soon as I get an answer

**Review by anxolin — APPROVED** [review]

**Review by anxolin — COMMENTED** [review]
Where is the URL for the API of the profile? (affiliate program)

**Comment by alfetopito** [comment]
> Where is the URL for the API of the profile? (affiliate program)

I completely forgot about that, my bad.

@matextrem could you update line 56 to https://api.cow.fi/affiliate/api

And also update the respective default on lines 39 - 41?

https://github.com/gnosis/cowswap/blob/2fc6c8187796903002f9dc6b7d1d30c33b0c95ea/src/custom/api/gnosisProtocol/api.ts#L30-L56

And the barn api urls wont be changing yet, could you please revert them?

File: .env.production [hunk 1]
```diff
@@ -56,18 +56,18 @@ REACT_APP_PATH_REGEX_ENS="/ipfs"
...
-#REACT_APP_API_URL_PROD_XDAI=https://protocol-xdai.gnosis.io/api
-#REACT_APP_API_URL_STAGING_MAINNET=https://protocol-mainnet.dev.gnosisdev.com/api
-#REACT_APP_API_URL_STAGING_RINKEBY=https://protocol-rinkeby.dev.gnosisdev.com/api
-#REACT_APP_API_URL_STAGING_XDAI=https://protocol-xdai.dev.gnosisdev.com/api
+#REACT_APP_API_URL_PROD_MAINNET=https://api.cow.fi/mainnet
+#REACT_APP_API_URL_PROD_RINKEBY=https://api.cow.fi/rinkeby
+#REACT_APP_API_URL_PROD_XDAI=https://api.cow.fi/xdai
+#REACT_APP_API_URL_STAGING_MAINNET=https://barn.api.cow.fi/mainnet
+#REACT_APP_API_URL_STAGING_RINKEBY=https://barn.api.cow.fi/rinkeby
+#REACT_APP_API_URL_STAGING_XDAI=https://barn.api.cow.fi/xdai
```
Comment (alfetopito) [inline .env.production:64]: Barn apis are not yet available and we'll release without them for now. Can you please revert them?

File: src/custom/api/gnosisProtocol/api.ts [hunk 2]
```diff
@@ -39,9 +39,9 @@ function getGnosisProtocolUrl(): Partial<Record<ChainId, string>> {
 
   // Production, staging, ens, ...
   return {
-    [ChainId.MAINNET]: process.env.REACT_APP_API_URL_PROD_MAINNET || 'https://protocol-mainnet.gnosis.io/api',
-    [ChainId.RINKEBY]: process.env.REACT_APP_API_URL_PROD_RINKEBY || 'https://protocol-rinkeby.gnosis.io/api',
-    [ChainId.XDAI]: process.env.REACT_APP_API_URL_PROD_XDAI || 'https://protocol-xdai.gnosis.io/api',
+    [ChainId.MAINNET]: process.env.REACT_APP_API_URL_PROD_MAINNET || 'https://api.cow.fi/mainnet',
+    [ChainId.RINKEBY]: process.env.REACT_APP_API_URL_PROD_RINKEBY || 'https://api.cow.fi/rinkeby',
+    [ChainId.XDAI]: process.env.REACT_APP_API_URL_PROD_XDAI || 'https://api.cow.fi/xdai',
```
Comment (alfetopito) [inline src/custom/api/gnosisProtocol/api.ts:44]:
```suggestion
    [ChainId.MAINNET]: process.env.REACT_APP_API_URL_PROD_MAINNET || 'https://api.cow.fi/mainnet/api',
    [ChainId.RINKEBY]: process.env.REACT_APP_API_URL_PROD_RINKEBY || 'https://api.cow.fi/rinkeby/api',
    [ChainId.XDAI]: process.env.REACT_APP_API_URL_PROD_XDAI || 'https://api.cow.fi/xdai/api',
```

File: .env.production [hunk 3]
```diff
@@ -56,18 +56,18 @@ REACT_APP_PATH_REGEX_ENS="/ipfs"
 #REACT_APP_SENTRY_AUTH_TOKEN='<sentry_auth_token>'
 
 # API
-#REACT_APP_API_URL_PROD_MAINNET=https://protocol-mainnet.gnosis.io/api
-#REACT_APP_API_URL_PROD_RINKEBY=https://protocol-rinkeby.gnosis.io/api
-#REACT_APP_API_URL_PROD_XDAI=https://protocol-xdai.gnosis.io/api
+#REACT_APP_API_URL_PROD_MAINNET=https://api.cow.fi/mainnet
+#REACT_APP_API_URL_PROD_RINKEBY=https://api.cow.fi/rinkeby
+#REACT_APP_API_URL_PROD_XDAI=https://api.cow.fi/xdai
```
Comment (alfetopito) [inline .env.production:61]: Same thing as on Henry's PR, sorry about that https://github.com/gnosis/gp-ui/pull/931/files#r779053245

```suggestion
#REACT_APP_API_URL_PROD_MAINNET=https://api.cow.fi/mainnet/api
#REACT_APP_API_URL_PROD_RINKEBY=https://api.cow.fi/rinkeby/api
#REACT_APP_API_URL_PROD_XDAI=https://api.cow.fi/xdai/api
```

**Review by alfetopito — APPROVED** [review]

**Pull request merged**